### PR TITLE
Miranda NG: update autoread implementation for new versions of the client

### LINKED ISSUE #9053: Miranda NG: Automatic messages reading is broken
Can someone please have a look at the Miranda32-App module or at whatever component that causes the following: If new messages arrive in an active session, they are no longer automatically read. As well as if you switch to another tab with unread messages, these unread messages are also not automatically read. It should not be that hard to fix, as it is an read only text box basically, where messages arrive. And one thing more: If you fix it, please make it as an AppModule or an addon that is also compatile with NVDA 2017.3 as I am here on Windows XP for good reasons.

### PR DESCRIPTION
**Pull request by codeofdusk**
### Link to issue number:
Fixes #9053

### Summary of the issue:
For the past several versions of Miranda NG, NVDA's message autoread support has been broken.
### Description of how this pull request fixes the issue:
New versions of the client seem to use a different `windowControlID` for the scrollbar in the message window. This pull request updates the `MESSAGEVIEWERS` constant in the app module to account for this change.
### Testing performed:
Sent several test messages to the #flood IRC channel on Freenode. Reading was functional but inconsistent (as expected).
### Known issues with pull request:
NVDA's autoread support is highly fragile and only functions intermitantly, see #1471.
### Change log entry:

 == New Features ==
- Restored NVDA's Miranda NG support for new versions of the client. (#9053)

### CODE REVIEW

File: source/appModules/miranda32.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #appModules/miranda32.py
 #A part of NonVisual Desktop Access (NVDA)
-#Copyright (C) 2006-2012 NVDA Contributors
+#Copyright (C) 2006-2019 NVDA Contributors, Bill Dengler
```
Comment (leonardder) [inline source/appModules/miranda32.py]: Please change this as follows:

```suggestion
#Copyright (C) 2006-2019 NV Access Limited, Aleksey Sadovoy, Peter Vágner, Joseph Lee, Bill Dengler
```

Also, please start the file with a new line, containing:
`# -*- coding: UTF-8 -*-`

**Review by LeonarddeR — APPROVED** [review]
Just one comment regarding the copyright header

**Comment by codeofdusk** [comment]
Done. Any suggestions for fixing the failing test cases?

**Comment by leonardder** [comment]
That's not a problem on your end and will be dealt with as part of #9358

File: source/appModules/miranda32.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
+# -*- coding: UTF-8 -*-
 #appModules/miranda32.py
 #A part of NonVisual Desktop Access (NVDA)
-#Copyright (C) 2006-2012 NVDA Contributors
+#Copyright (C) 2006-2019 NV Access Limited, Aleksey Sadovoy, Peter Vágner, Joseph Lee, Bill Dengler
```
Comment (michaelDCurran) [inline source/appModules/miranda32.py]: Just curious, but how did you work out that Aleksey Sadovoy, Peter Vágner and Joseph Lee contributed to this file? Did you look at the repository history?

**Comment by leonardder** [comment]
To find out all the contributors, I ran a git blame over the whole file.

**Review by michaelDCurran — APPROVED** [review]

**Pull request merged**